package.json

```diff
@@ -30,12 +30,16 @@
     "eslint-plugin-import": "2.8.0",
     "eslint-plugin-prettier": "2.5.0",
     "eslint-plugin-react": "7.6.1",
+    "fusion-core": "0.3.1",
     "flow-bin": "0.64.0",
     "nyc": "11.4.1",
     "prettier": "1.10.2",
     "tape-cup": "4.7.1",
     "unitest": "2.1.1"
   },
+  "peerDependencies": {
+    "fusion-core": "0.3.1"
```

Comment:
Probably should be a loose peerDependency version?

Reply:
+1, looks like we're generally fine with carot ranges here.